stdlib/messaging/rabbitmq/src/main/java/org/ballerinalang/messaging/rabbitmq/util/ListenerUtils.java

```diff
@@ -0,0 +1,267 @@
+            prefetchCount = serviceConfig.getIntValue(RabbitMQConstants.ALIAS_PREFETCH_COUNT);
+        }
+        boolean isValidPrefetchSize = serviceConfig.getIntValue(RabbitMQConstants.ALIAS_PREFETCH_SIZE) != null;
+        try {
+            if (isValidPrefetchSize) {
+                channel.basicQos(Math.toIntExact(serviceConfig.getIntValue(RabbitMQConstants.ALIAS_PREFETCH_SIZE)),
+                        Math.toIntExact(prefetchCount), false);
+            } else {
+                channel.basicQos(Math.toIntExact(prefetchCount));
+            }
+        } catch (IOException | ArithmeticException exception) {
+            throw new RabbitMQConnectorException("An error occurred while setting the basic QoS settings; "
+                    + exception.getMessage(), exception);
+        }
+    }
+
+    private static boolean isStarted() {
+        return started;
+    }
+
+    private static boolean getAckMode(ObjectValue service) {
+        boolean autoAck;
+        MapValue serviceConfig = (MapValue) service.getType().getAnnotation(RabbitMQConstants.PACKAGE_RABBITMQ,
+                RabbitMQConstants.SERVICE_CONFIG);
+        @SuppressWarnings(RabbitMQConstants.UNCHECKED)
+        String ackMode = serviceConfig.getStringValue(RabbitMQConstants.ALIAS_ACK_MODE);
+        switch (ackMode) {
+            case RabbitMQConstants.AUTO_ACKMODE:
+                autoAck = true;
+                break;
+            case RabbitMQConstants.CLIENT_ACKMODE:
+                autoAck = false;
+                break;
+            default:
+                throw RabbitMQUtils.returnErrorValue("Unsupported acknowledgement mode");
+        }
+        return autoAck;
+    }
+
+    public static Object stop(ObjectValue listenerObjectValue) {
+        Channel channel = (Channel) listenerObjectValue.getNativeData(RabbitMQConstants.CHANNEL_NATIVE_OBJECT);
+        if (channel == null) {
+            return RabbitMQUtils.returnErrorValue("ChannelListener is not properly initialised.");
+        } else {
+            try {
+                Connection connection = channel.getConnection();
+                channel.close();
+                connection.close();
+            } catch (IOException | TimeoutException exception) {
+                return RabbitMQUtils.returnErrorValue(RabbitMQConstants.CLOSE_CHANNEL_ERROR
+                        + exception.getMessage());
+            }
+        }
+        return null;
+    }
+
+    public static Object abortConnection(ObjectValue listenerObjectValue) {
+        Channel channel = (Channel) listenerObjectValue.getNativeData(RabbitMQConstants.CHANNEL_NATIVE_OBJECT);
+        if (channel == null) {
+            return RabbitMQUtils.returnErrorValue("ChannelListener is not properly initialised.");
+        } else {
+            Connection connection = channel.getConnection();
+            connection.abort();
+        }
+        return null;
+    }
+
+    static {
+        console = System.out;
```

Comment:
Can do this at the variable declaration itself. no need of a static block?

Reply:
This is to avoid the checkstyle failure.